maro/simulator/scenarios/data_center/topologies/azure.2019.original/config.pm.size.yml

```diff
@@ -0,0 +1,6 @@
+N1:
```

Comment:
What's the meaning of N1?

Reply:
SKU_List:
  - XXX
  - YYY
PM_NODES:
  XXX:
    Amount:
    CPU:
    Memory:
    Power_Curve:

Reply:
see the raw mode of last comments